src/helper/json_helper.hpp

```diff
@@ -28,12 +30,12 @@ class string_buffer {
     void ShrinkToFit() { buffer_.shrink_to_fit(); }
     void Reserve(size_t count) { buffer_.reserve(count); }
 
-    [[nodiscard]] const Ch* GetString() const { return buffer_.c_str(); }
+    [[nodiscard]] const Ch *GetString() const { return buffer_.c_str(); }
```

Comment:
Not a huge fan of having the pointer or ampersand on the function itself rather than the type when its a return, but the only options I've seen would also impact variable definitions... Any ideas?